cmd/kubeadm/app/apis/bootstraptoken/v1/utils.go

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
these and the tests in utils_test.go are copied from the versioned / internal bootstrap* files.